src/ui/components/DashKit/plugins/Title/Title.tsx

```diff
@@ -97,6 +108,60 @@ const titlePlugin = {
             data.text,
         ]);
 
+        const showAnchor = !disableHashNavigation && !props.editMode;
+        const withInlineAnchor = showAnchor && isInlineAnchor;
+        const withAbsoluteAnchor = showAnchor && !isInlineAnchor;
+
+        const calculateAnchor = React.useCallback(() => {
+            if (showAnchor && contentRef.current && rootNodeRef.current) {
+                const contentHeight = contentRef.current.getBoundingClientRect().height || 0;
```

Comment:
`.current.getBoundingClientRect()` - let's save this calculations once and use their values as vars.